bin/station.js

```diff
@@ -1,36 +1,15 @@
 #!/usr/bin/env node
 
-import { join, dirname } from 'node:path'
-import { homedir } from 'node:os'
+import * as commands from '../commands/index.js'
 import fs from 'node:fs/promises'
-import * as saturnNode from '../lib/saturn-node.js'
-import { fileURLToPath } from 'node:url'
-import { createLogStream } from '../lib/log.js'
-
-const {
-  FIL_WALLET_ADDRESS,
-  XDG_STATE_HOME = join(homedir(), '.local', 'state')
-} = process.env
-
-if (!FIL_WALLET_ADDRESS) {
-  console.error('FIL_WALLET_ADDRESS required')
-  process.exit(1)
-}
-
-const paths = {
-  metrics: join(XDG_STATE_HOME, 'filecoin-station', 'logs', 'metrics.log'),
-  moduleStorage: join(XDG_STATE_HOME, 'filecoin-station', 'modules'),
-  moduleLogs: join(XDG_STATE_HOME, 'filecoin-station', 'logs', 'modules'),
-  moduleBinaries: join(dirname(fileURLToPath(import.meta.url)), '..', 'modules')
-}
+import { join } from 'node:path'
+import { paths } from '../lib/paths.js'
 
 await fs.mkdir(join(paths.moduleStorage, 'saturn-L2-node'), { recursive: true })
 await fs.mkdir(paths.moduleLogs, { recursive: true })
 
-await saturnNode.start({
-  FIL_WALLET_ADDRESS,
-  storagePath: join(paths.moduleStorage, 'saturn-L2-node'),
-  binariesPath: paths.moduleBinaries,
-  metricsStream: createLogStream(paths.metrics),
-  logStream: createLogStream(join(paths.moduleLogs, 'saturn-L2-node.log'))
-})
+if (process.argv.includes('metrics')) {
+  await commands.metrics()
+} else {
+  await commands.station()
+}
```

Comment:
Have you considered using a library to parse CLI args? Node.js is adding built-in support, but it's experimental, so we should probably use something else like minimise or yargs. 

https://oclif.io seems to be a more modern option.

Reply:
I have thought about it, but for now that would be overkill imo. The simple logic here works and doesn't need (a tree of) dependencies. I'd suggest to add a lib like this (would be happy to try oclif, or the trusted minimist/yargs) once we see the current approach doesn't scale or causes bugs